Assignment2-Python/stringCompression.py

```diff
@@ -0,0 +1,32 @@
+def compress_string(string):
+    if not string:
+        return ''
+
+    compress = []
+    count = 1
+
+    for i in range(1, len(string)):
```

Comment:
This might be slightly cleaner as a loop over `range(len(string) - 1)` only because you use `i - 1` once more than `i`.